front/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,25 @@
+name: Playwright Tests
+on:
+  pull_request:
+    branches: [ dev, main ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - name: Install dependencies
+      run: yarn
+    - name: Install Playwright Browsers
+      run: yarn playwright install --with-deps
+    - name: Run Playwright tests
+      run: yarn playwright test
+    - uses: actions/upload-artifact@v3
+      if: always()
+      with:
+        name: playwright-report
+        path: playwright-report/
+        retention-days: 30
```

Comment:
Shouldn't this part of `.github/workflows/front.yml` , not `front/.github/workflows/playwright.yml`?

Reply:
It was moved to integration tests.